test/topology/test_aggregation.py

```diff
@@ -18,6 +18,8 @@
 from test.topology.util import new_test_keyspace, new_test_table
 
 logger = logging.getLogger(__name__)
+pytestmark = pytest.mark.usefixtures("prepare_3_nodes_cluster")
```

Comment:
Note that by putting this pytestmark trickery right next to the "logger" variable, you make it even more opaque - it looks like code setting a variable, and then you put it next to other code which sets a variable to hide it even better ;-)

Personally, I would have given this important but hard to understand and (to be honest) opaque trick its own special place not stuck to other variable settings, and even preceded it with a comment. But I don't insist.